examples/renderer_utils.py

```diff
@@ -179,7 +180,7 @@ def Vector2(x, y):
 
 @ti.func
 def inside_taichi(p_):
```

Comment:
Use `taichi_logo` instead?

Reply:
Why not :> When I reviewed this part I assume it kept the old way instead of the new `taichi_logo`...well, because this PR is still undergoing?

:> The only reason I can imagine.

Will come back to look again if any updates happen.

Reply:
Moved now :)